packages/@aws-cdk/aws-logs/lib/log-retention-provider/index.ts

```diff
@@ -51,6 +51,35 @@ async function createLogGroupSafe(logGroupName: string, region?: string, options
   } while (true); // exit happens on retry count check
 }
 
+//delete a log group
+async function deleteLogGroup(logGroupName: string, region?: string, options?: SdkRetryOptions) {
```

Comment:
After thinking about it, I think this function is actually fine as is. Go ahead and disregard my previous feedback here.